mkdocs.yml

```diff
@@ -134,6 +134,7 @@ pages:
       - Configuration Locations: vm-config.md
       - Using Logs: job-logs.md
       - Instance Metadata: instance-metadata.md
+    - Rotating NATS Certificates: nats-certificate-rotation.md
```

Comment:
Note to selves; when we have more rotation docs, we may want a "Rotating Credentials" or general "Credentials" section.